src/components/LatestZephyrManifest.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import CodeBlock from "@theme/CodeBlock";
+
+export function ZephyrManifest() {
+    const [latestTag, setLatestTag] = useState("");
```

Comment:
```suggestion
    const FALLBACK_LATEST_TAG = "0.43.3"
    const [latestTag, setLatestTag] = useState(FALLBACK_LATEST_TAG);
```

Then, only `setLatestTag` in the `try` branch if there is one:

```jsx
const tag = data?.tag_name;
if (tag) {
  setLatestTag(tag);
}
```